collectionize.js

```diff
@@ -26,14 +26,16 @@
   function nativeEach(obj, cb) {
     if (obj && _.isFunction(cb)) {
       if (_.isPlainObject(obj)) {
-        return eachObject(obj, cb);
-      } else if (_.isArray(obj) || _.isString(obj)) {
-        return eachArray(obj, cb);
+        eachObject(obj, cb);
+      } else if (_.isArray(obj)) {
+        obj.forEach(cb);
```

Comment:
the native `forEach` does not have a way to cancel iteration, but I have checked the code and I don't believe we use this feature.